### ISSUE TITLE
Mention Raspberry Pi dependency in README.md

### ISSUE THREAD

**Issue by davecoutts** [Brandawg93/Pi-Hole-Monitoring#3]
Can i suggest adding to the README.md that the docker-compose.yml configuration expects Pi-hole to be running on a Raspberry Pi?

Compose up will fail on non Raspberry Pi installations with the error below as the docker-compose.yml telegraf section expects to mount /dev/vchiq and /dev/vcsm , which are devices associated with the VideoCore interface of the Raspberry Pi.

ERROR: for telegraf Cannot start service telegraf: error gathering device information while adding custom device "/dev/vchiq": no such file or directory

**Comment by redsquirrelperson** [comment]
Assuming this is not fixable - I don't have this running on a Raspberry PI, how do I uninstall  what has already been installed.

**Comment by davecoutts** [comment]
To get it running rather than deleting the installation, you could comment out /dev/vchiq and /dev/vcsm in the telegraf section of docker-compose.yml. See snippet below.
And then run docker-compose up -d again.

This is what I have done for a Pi-hole instance running on Ubuntu Server 18.04 on VMware.
At a glance all I am missing in the Grafana metrics is system measures like CPU temperature.
The Pi-hole metrics work fine.

```yaml
  telegraf:
    image: telegraf:latest
    container_name: telegraf
    depends_on:
      - influxdb
    networks:
      - influxdb
#    devices:
#      - /dev/vchiq
#      - /dev/vcsm
    volumes:
      - ./telegraf:/etc/telegraf
      - /opt/vc:/opt/vc
    restart: always
```

**Comment by redsquirrelperson** [comment]
Thanks Dave, all up and running now

**Comment by Brandawg93** [comment]
I’m looking into support for multiple devices. For now, commenting those out is the best option for Ubuntu. Thanks Dave.

**Comment by Brandawg93** [comment]
I’ve added this to the [README](https://github.com/Brandawg93/Pi-Hole-Monitoring/blob/master/README.md#known-issues). Thanks for the heads up. 

Edit: Fixed in 8e6928fb9fc8f94595e3ceb0a248e06c6c9b13ca
